java/src/jmri/jmrix/openlcb/swing/monitor/MonitorPane.java

```diff
@@ -105,6 +105,17 @@ void format(String prefix, boolean extended, int header, int len, int[] content)
                     formatted = prefix + ": (Start of Datagram)";
                 } else if ((header & 0x0F000000) == 0x0C000000) {
                     formatted = prefix + ": (Middle of Datagram)";
+                } else if (((header & 0xFFFF0000) == 0x19A00000) && (content.length > 0)) {
```

Comment:
You should take one more nibble and ignore the top bit:

header & 0x0FFFF000 == 0x09a08000